Gems/ScriptEvents/Code/Include/ScriptEvents/ScriptEventDefinition.h

```diff
@@ -151,7 +151,7 @@ namespace ScriptEvents
         
         const AZStd::vector<Method>& GetMethods() const { return m_methods; }
 
-        AZStd::string_view GetLabel() const { return GetName(); }
+        AZStd::string_view GetLabel() const { return m_name.Get<AZStd::string>()->c_str(); }
```

Comment:
suggestion: You should be able to use the string size here as a small optimization (so we don't have to parse the length of the string from `c_str()`)

e.g.:

```c++
class Test {
    std::string m_string = "hello";
public:
    std::string_view get_string() const { return {m_string.data(), m_string.size()}; } // this version
};
```

Reply:
Thanks Tom, change made.